virt/logging_events_monitoring/virt-monitoring-vm-health.adoc

```diff
@@ -18,8 +18,6 @@ include::modules/virt-define-http-liveness-probe.adoc[leveloffset=+1]
 
 include::modules/virt-define-guest-agent-ping-probe.adoc[leveloffset=+1]
 
-include::modules/virt-template-vm-probe-config.adoc[leveloffset=+1]
-
```

Comment:
Question: did you mean to delete this module?

Reply:
I did! Mentioned in the JIRA but not here. Thanks!